pkg/resource/cluster/cluster_read.go

```diff
@@ -64,6 +70,10 @@ func (m *Manager) readCluster(d ReadWrite) error {
 			}
 		}
 	}
+	//Do not use d.Set , only for debug.
```

Comment:
```suggestion
```

Can we remove this comment as (I thought) calling `d.Set` on Read should be avoided as it resulted in nasty errors/warnings?